### ISSUE TITLE
Suppression du paramètre ``get_role`` du décorateur ``check_cruved_scope``

### ISSUE THREAD

**Issue by bouttier** [PnX-SI/GeoNature#2162]
Le paramètre ``get_role=True`` permet aux vues décorées par ``check_cruved_scope`` de recevoir un argument supplémentaire ``info_role``. Cette objet est un petit peu particulier puisqu’il s’agit d’une instance du modèle ``VUsersPermissions`` correspondant à la vue SQL ``v_roles_permissions``. Dans l’objectif de supprimer cette vue, le paramètre ``get_role`` doit être supprimé.

L’objet ``info_role`` est majoritairement utilisé de 2 manières :
- Récupération de l’id du rôle : ``info_role.id_role`` → utilisez ``g.current_user.id_role``
- Récupération du scope : ``int(info_role.value_filter)`` → passez le paramètre ``get_scope=True`` à ``check_cruved_scope`` et votre vue recevra un paramètre ``scope`` (qui est déjà un int !)

**Comment by jpm-cbna** [comment]
Pour `g.current_user.id_role` est ce utilisable dans les modules externes ?
Comment importe t on `g` ?

**Comment by bouttier** [comment]
Oui, ``g`` est utilisable partout, c’est l’[Application Context](https://flask.palletsprojects.com/en/2.2.x/appcontext/).
Import : ``from flask import g``
Définition de ``current_user`` dans GeoNature via [``before_request``](https://flask.palletsprojects.com/en/2.2.x/api/#flask.Flask.before_request) : https://github.com/PnX-SI/GeoNature/blob/2.10.3/backend/geonature/app.py#L137
Peut valoir ``None`` pour les routes non authentifiées, mais est correctement définie pour toutes les routes protégé par ``@check_cruved_scope`` ou ``@login_required``

**Comment by camillemonchicourt** [comment]
Usage de ``get_role`` supprimé dans les modules Synthèse et Validation : https://github.com/PnX-SI/GeoNature/commit/57361cdb64fcc1b5a216ecc581f9bacc2057e8f5

**Comment by bouttier** [comment]
Paramètre ``get_role`` supprimé dans develop

**Comment by camillemonchicourt** [comment]
Supprimé dans la 2.12.0
